src/atomate2/vasp/sets/mp.py

```diff
@@ -131,19 +134,4 @@ def get_incar_updates(
         dict
             A dictionary of updates to apply.
         """
-        bandgap = self.bandgap_override or bandgap or 0
-
-        if bandgap < self.bandgap_tol:  # metallic
-            return {"KSPACING": 0.22, "ISMEAR": 2, "SIGMA": 0.2, "GGA": None}
-
-        rmin = 25.22 - 2.87 * bandgap
-        kspacing = 2 * np.pi * 1.0265 / (rmin - 1.0183)
-        return {
-            "LWAVE": True,
-            "LCHARG": True,
-            "KSPACING": kspacing if 0.22 < kspacing < 0.44 else 0.44,
-            "ISMEAR": 0,
-            "SIGMA": 0.05,
-            "GGA": None,  # VASP 6.4+ errors if both GGA and METAGGA tag are set
-            # GGA tag might come from prev job INCAR inheritance, unset it to be safe
-        }
+        return {"LCHARGE": True, "LWAVE": True}
```

Comment:
@utf Is there a better way to override the `"LCHARG": False` from `RelaxSetGenerator` here?

Reply:
No, this is the recommended way to do it. All the particular settings should always be specified in one place in the input set. `get_incar_updates` so it is always clear where to find them